src/main/kotlin/com/ampnet/auditornode/model/EthereumAddress.kt

```diff
@@ -0,0 +1,3 @@
+package com.ampnet.auditornode.model
+
+inline class EthereumAddress(val value: String)
```

Comment:
Indeed, however the current version of Kotlin used in this project (1.4.0) does not seem to support value classes. The project is currently on 1.4.0 because of dependency issues, this will be fixed at the later time... I will leave TODOs to convert inline classes to value classes in that case.